### PR TITLE
Added trial token status code information to OT troubleshooting page

### PR DESCRIPTION
**Pull request by samdutton**
NB: I'm having problems building locally, so haven't tested layout yet.

### CODE REVIEW

File: site/en/docs/web-platform/origin-trial-troubleshooting/index.md [hunk 1]
```diff
@@ -160,6 +161,65 @@ Application panel. You may need to reload Chrome DevTools (not the page).
...
+include the scheme, hostname, or port.<br>
+[Source code](https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/common/origin_trials/trial_token.cc;drc=610603f89f0dd4da794848e4f8670a179efbcf38;l=262)
+
+* **InvalidSignature**: The token has an invalid or malformed signature.<br>
+[Source code](https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/common/origin_trials/trial_token_validator_unittest.cc;l=105)
+
+* **Malformed**: Token is malformed and could not be parsed.<br>
+[Source code](https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/common/origin_trials/trial_token.cc;l=88)
+
+{% Aside 'important' %}
```
Comment (jeffposnick) [inline site/en/docs/web-platform/origin-trial-troubleshooting/index.md:201]: Please change this to `'caution'`, not `'important'`, as we don't have any i18n for the phrase `'important'`.

**Review by jeffposnick — CHANGES_REQUESTED** [review]

File: site/en/docs/web-platform/origin-trial-troubleshooting/index.md (hunk 1 above)
Comment (samdutton) [reply site/en/docs/web-platform/origin-trial-troubleshooting/index.md:201]: Ah — thanks.

Comment (samdutton) [reply site/en/docs/web-platform/origin-trial-troubleshooting/index.md]: Thanks! Changed.

**Comment by samdutton** [comment]
Hi @jeffposnick or @heyawhite — could you approve this (or not :)? Thanks.

**Review by heyawhite — APPROVED** [review]

**Pull request merged**